config/helm/chart/default/templates/Common/edge-connect/clusterrole-edgeconnect.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if and .Values.rbac.edgeConnect.create (eq (include "dynatrace-operator.openshiftOrOlm" .) "true") }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: dynatrace-edgeconnect
+  namespace: {{ .Release.Namespace }}
+  {{- if .Values.rbac.edgeConnect.annotations }}
+  annotations:
+    {{- toYaml .Values.rbac.edgeConnect.annotations | nindent 4 }}
+  {{- end }}
+  labels:
+    {{- include "dynatrace-operator.operatorLabels" . | nindent 4 }}
+rules:
+- apiGroups:
+  - security.openshift.io
```

Comment:
It's formatted differently than another clusterRoless, https://github.com/dynatrace/dynatrace-operator/blob/4d01742c31c7cde41484d0b6ca92d060567f2240/config/helm/chart/default/templates/Common/kubernetes-monitoring/clusterrole-kubernetes-monitoring.yaml#L39-L44